### ISSUE TITLE
CC disabled when API field missing in config, pool-side TLS not working

### ISSUE THREAD

**Issue by djfinch** [Bendr0id/xmrigCC-amd#1]
Hi Ben,

Thank you, currently testing on OSX. Two issues spotted:

1. pool-side TLS is not working ATM, "use-tls": false option in pool seems to be ignored. TLS against CC-Server works. Yes, miner-side was compiled with TLS support.

2. I don't want to use build-in API so below field was removed from config (cause API port is "0" so I thought it's not needed there). However, after that CC-Server connection is not working anymore. Revert back to below fixed this. Yes, miner-side was compiled with API support.
```
  "api": {
    "port": 0,
    "access-token": null,
    "worker-id": null,
    "ipv6": false,
    "restricted": true
  },
```
WITHOUT API FIELD
![image](https://user-images.githubusercontent.com/13107713/42845719-1dec057e-8a17-11e8-99c8-f0d9fd2bbbc9.png)

WITH API FIELD
![image](https://user-images.githubusercontent.com/13107713/42845600-bbf8f0c0-8a16-11e8-8f96-30fe97c3c39e.png)

**Comment by Bendr0id** [comment]
Thanks for testing.

That TLS for stratum/pool isn't working is a known issue. It's just not implemented yet.

I'll disable the API support like I did on XMRigCC, then this issue should be gone aswell.

**Comment by Bendr0id** [comment]
Should be disabled in master now

**Comment by Bendr0id** [comment]
Released in 1.6.5_beta3
